command-control-rest-daemon.py

```diff
@@ -50,10 +50,13 @@ def execute_command(cmd):
     print("execute: {}".format(cmd))
     args = cmd['cmd'].split()
     use_shell = False
+    stdout = None
     if 'shell' in cmd and cmd['shell'] == True:
         args = cmd['cmd']
         use_shell = True
-    process = subprocess.Popen(args, shell=use_shell)
+    if 'silent' in cmd and cmd['silent'] == True:
+        stdout = subprocess.DEVNULL
+    process = subprocess.Popen(args, stdout=stdout, shell=use_shell)
```

Comment:
what happens if silent is not specified/false: stdout will be None and then?

Reply:
stdout=None leads to the default behaviour which is to not redirect:

"stdin, stdout and stderr specify the executed program’s standard input, standard output and standard error file handles, respectively.
"With the default settings of None, no redirection will occur; the child’s file handles will be inherited from the parent."
(from docs.python.org)